backend/app/routers/files.py

```diff
@@ -43,6 +43,57 @@
 router = APIRouter()
 security = HTTPBearer()
 
+
+async def _resubmit_file_extractors(
+    file_id: str,
+    credentials: HTTPAuthorizationCredentials = Security(security),
+    db: MongoClient = Depends(dependencies.get_db),
+    rabbitmq_client: BlockingChannel = Depends(dependencies.get_rabbitmq),
+):
+    """This helper method will check metadata. We get the extractors run from metadata from extractors.
+    Then they are resubmitted. At present parameters are not stored. This will change once Jobs are
+    implemented.
+
+        Arguments:
+        file_id: Id of file
+        credentials: credentials of logged in user
+        db: MongoDB Client
+        rabbitmq_client: Rabbitmq Client
+
+    """
+    if (file := await db["files"].find_one({"_id": ObjectId(file_id)})) is not None:
+        file_out = FileOut.from_mongo(file)
+        query = {"resource.resource_id": ObjectId(file_id)}
```

Comment:
this will match all metadata regardless of resource.version, so any previous file version metadata will also be deleted. is this the intent?

Reply:
I notice that MetadataOut doesn't seem to have the version. I will add that and then check to make sure this is the latest.

Reply:
Added that, just need to test to make sure it works.